packages/ui/src/views/repo/pull-request/compare/pull-request-compare-page.tsx

```diff
@@ -286,21 +284,27 @@ export const PullRequestComparePage: FC<PullRequestComparePageProps> = ({
         {prBranchCombinationExists && (
           <Layout.Horizontal className="mt-4 items-center justify-between rounded-md border border-borders-1 bg-background-2 p-4">
             <div className="flex items-center gap-x-1.5">
-              <Icon name="compare" size={14} className="text-icons-success" />
-              <div className="flex gap-x-1.5">
-                {/* TODO: add the name of the PR instead this placeholder */}
-                <p className="text-14 text-foreground-1">PR for this combination of branches already exists.</p>
-                <span className="text-foreground-4">{`#${prBranchCombinationExists}`}</span>
+              <div>
+                <Layout.Horizontal className="items-center">
+                  <Icon name="compare" size={14} className="text-icons-success" />
+                  <div className="flex gap-x-1">
+                    {/* TODO: add the name of the PR instead this placeholder */}
+                    <p className="text-14 text-foreground-1">{prBranchCombinationExists.title}</p>
+                    <span className="text-foreground-4">{`#${prBranchCombinationExists.number}`}</span>
+                  </div>
+                </Layout.Horizontal>
+
+                <p className="text-14  text-foreground-2">{prBranchCombinationExists.description}</p>
               </div>
             </div>
-            <Button onClick={() => navigate(`../${prBranchCombinationExists}/conversation`)}>
+            <Button onClick={() => navigate(`../${prBranchCombinationExists.number}/conversation`)}>
               {t('views:pullRequests.compareChangesViewPRLink', 'View pull request')}
             </Button>
           </Layout.Horizontal>
         )}
         {isBranchSelected ? (
           <Layout.Vertical className="mt-10">
-            <Tabs variant="tabnav" defaultValue={!prBranchCombinationExists ? 'overview' : 'commits'}>
+            <Tabs variant="tabnav" value={prBranchCombinationExists?.number ? 'commits' : 'overview'}>
```

Comment:
probably should do a `!isNaN` if the intention if to check if it's a number.

Reply:
ah it can actually just be the object?

Reply:
in that case, it should be fine, but typically you'd want to rely on a key from an object to check ternary condition and not the whole object.

Reply:
ok